miso-web/src/main/webapp/scripts/bulk.js

```diff
@@ -0,0 +1,1053 @@
+BulkUtils = (function($) {
+
+  /*
+   * BulkTarget structure: {
+   *   getSaveUrl: required function() returning url for bulk save. POST is used for create and PUT for edits
+   *   getUserManualUrl: optional (recommended) function() returning url of specific user manual page to set on 'Help' link
+   *   getCustomActions: optional function() returning array of Custom Actions (see below) to make available while editing
```

Comment:
> while editing

On the bulk edit page while editing? I'm not sure why I find this weird.

Reply:
Updated. Also fixed incomplete documentation of the API object's `updateField` options